daemon/control.c

```diff
@@ -542,6 +548,58 @@ control_handle_cmd_list_guestos_configs(UNUSED const ControllerToDaemon *msg, in
 	mem_free0(results);
 }
 
+/**
+ * Handles list_guestos_configs cmd.
+ * Used in both priv and unpriv control handlers.
+ */
+static void
+control_handle_cmd_retrieve_logs(UNUSED const ControllerToDaemon *msg, int fd)
+{
+	control_message_t message = CONTROL_RESPONSE_CMD_FAILED;
+	char *data_logs_dir = "/data/logs/";
```

Comment:
A define LOGFILE_DIR "/data/logs" exists in cmld.h, use this to be compatible with lsb path, which is overwritten in the Makefile_lsb

Reply:
LOGFILE_DIR is now used instead of "/data/logs"